falcon/api.py

```diff
@@ -391,12 +391,15 @@ def add_static_route(self, prefix, directory, downloadable=False):
             downloadable (bool): Set to ``True`` to include a
                 Content-Disposition header in the response. The "filename"
                 directive is simply set to the name of the requested file.
+            default_filename (str): Default filename used when the requested file
```

Comment:
It's an error on my part because I thought that join would merge two absolute path, but it does not. 
I meant to only allow files inside the given directory.

Do you think it's best to allow files that are outside of the given directory?